tools/docker/compose

```diff
@@ -103,9 +103,7 @@ case "$1" in
   test)
     $test build $DOCKER_COMPOSE_BUILD_OPTS
     $deps up --exit-code-from wait-db wait-db
-    set +e
```

Comment:
I assume this was the part that was causing it to pass even though it should have failed